### ISSUE TITLE
no ticks nor frame on small maps 

### ISSUE THREAD

**Issue by zuliani71** [GenericMappingTools/gmt#1367]
**Description of the problem**
I can't correctly plot a psbasemap if the range of the map is less than a 1 degree. The ps file just includes the grid tracks without the frame and the tick marks.

**Full script that generated the error**

```
psbasemap -R13.252/13.254/45.962/45.963 -Jm1:2000 -Ba2sf1sg2s/a2sf1sg2s -V> ./out.ps
```

**Full error message**

```
PASTE ERROR MESSAGE HERE
```

**System information**

* Operating system: macOS Mojave 10.14.5
* Version of GMT:
5.4.5

**Comment by seisman** [comment]
You're using old GMT4 syntax for -B option. For GMT5, you can use:

```
gmt psbasemap -R13.252/13.254/45.962/45.963 -Jm1:2000 -Bxa2sf1sg2s -Bya2sf1sg2s -V> out.ps
```

**Comment by zuliani71** [comment]
Hello, now it works, thank you so much. Bye

**Comment by joa-quim** [comment]
But we said recently that GMT4 syntax is supposed to keep working, so this is an issue.

**Comment by PaulWessel** [comment]
Fixed by #1382.
